buildenv/jenkins/jobs/infrastructure/copyrightCheck.groovy

```diff
@@ -57,15 +57,15 @@ timeout(time: 6, unit: 'HOURS') {
                     // Set a different Copyright regex depending on the Repo the PR is from
                     REGEX = "\'Copyright \\(c\\) ([0-9]{4}), ${DATE_YEAR}\'"
 
-                    WHITE_LIST = []
-                    if (fileExists("${WORKSPACE}/.copyright_whitelist")) {
-                        WHITE_LIST = readFile '.copyright_whitelist'
-                        WHITE_LIST = WHITE_LIST.tokenize("\n")
+                    ALLOW_LIST = []
+                    if (fileExists("${WORKSPACE}/.copyrightIgnore")) {
+                        ALLOW_LIST = readFile '.copyrightIgnore'
+                        ALLOW_LIST = ALLOW_LIST.tokenize("\n")
```

Comment:
I suggest this would be easier to follow if the variable were `IGNORE_LIST` to be consistent with the name of the file it is from (which should be `.copyrightignore`).